src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetUptime.cs

```diff
@@ -0,0 +1,71 @@
+/********************************************************************++
+Copyright (c) Microsoft Corporation.  All rights reserved.
+--********************************************************************/
+
+using System;
+using System.Management.Automation;
+using System.Diagnostics;
+using System.Management.Automation.Internal;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// This class implements Get-Uptime
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, "Uptime", DefaultParameterSetName = "Timespan", HelpUri = "")]
```

Comment:
Good question! I searched the code base and it seems the `ParameterSetName` information is not really used anywhere today. But having more metadata is always a good thing and it might be useful someday, so yes let's include the parameter set names :)

As for the namespaces, my recommendation is to not include unnecessary namespace prefix unless it will result in ambiguity otherwise.

We will need to request a help uri for this new cmdlet. @Francisco-Gamino, is it OK to leave it as an empty string at this moment? Will this cause test failures in our nightly build?